src/test/groovy/org/fundacionjala/gradle/plugins/enforce/utils/salesforce/FileValidatorTest.groovy

```diff
@@ -153,8 +152,8 @@ class FileValidatorTest extends Specification {
             ClassifiedFile classifiedFile = FileValidator.validateFiles(SRC_PATH, allFiles)
         then:
             classifiedFile.invalidFiles.sort() == mapExpected[Constants.INVALID_FILE_BY_FOLDER].sort()
-            classifiedFile.notFoundFiles.sort() == mapExpected[Constants.DOES_NOT_EXIST_FILES].sort()
-            classifiedFile.filesWithoutXml.sort() == mapExpected[Constants.FILE_WITHOUT_XML].sort()
+            classifiedFile.filesNotFound.sort() == mapExpected[Constants.DOES_NOT_EXIST_FILES].sort()
+            classifiedFile.filesWithoutMetadata.sort() == mapExpected[Constants.FILE_WITHOUT_XML].sort()
```

Comment:
You need to rename also these constants
Constants.FILE_WITHOUT_XML => Constants.FILE_WITHOUT_METADATA
Constants.DOES_NOT_EXIST_FILES => Constants.FILES_NOT_FOUND